src/index.js

```diff
@@ -26,6 +27,11 @@ const browser = require('./browser-utils/index')
 // create a File from a pathOrDescriptor
 function open(pathOrDescriptor, { basePath, format } = {}) {
   let descriptor = null
+
+  if (browser.checkFileFromBrowser(pathOrDescriptor)) {
```

Comment:
Just to keep the same style with the other functions, maybe it is worth to have another naming like `isFileFromBrowser`?

Reply:
@kmanaseryan sure! let me improve it.